templates/README.md

```diff
@@ -0,0 +1,29 @@
+# <Repository Name>
+
+<!-- Repo Badges for: Github Project, Slack, License-->
+
+[![GitHub](https://img.shields.io/badge/project-Data_Together-487b57.svg?style=flat-square)](http://github.com/datatogether)
+[![Slack](https://img.shields.io/badge/slack-Archivers-b44e88.svg?style=flat-square)](https://archivers-slack.herokuapp.com/)
+[![License](https://img.shields.io/github/license/mashape/apistatus.svg)](./LICENSE) 
+
+[1-3 sentence description of repository contents]
+
+## License & Copyright
+
+[Modelled on [project guidelines template](./PROJECT.md#license--copyright-readme-block) ]
+
+## Getting Involved
+
+We would love involvement from more people! If you notice any errors or would like to submit changes, please see our [Contributing Guidelines](./CONTRIBUTING.md).
+
```

Comment:
Can we add an optional section here titled "Installation", and have it be instructions for _using_ the service that should provide hello-world grade instructions & includes an example of what the user should see when it works?